frontend/src/Frontend/UI/Dialogs/Send.hs

```diff
@@ -118,12 +123,17 @@ sendDeploy
   -> ([Either Text NodeInfo], PublicMeta, NetworkName) -- ^ Misc network information
   -> Workflow t m (mConf, Event t ())
 sendDeploy _model sender gasPayer recipient amount (nodeInfos, publicMeta, networkId) = Workflow $ do
-  let code = T.unwords
-        [ "(coin.transfer"
-        , tshow $ unAccountName $ _account_name sender
-        , tshow $ unAccountName $ _kadenaAddress_accountName recipient
-        , tshow amount
-        , ")"
+  let code = T.unwords $ catMaybes
+        [ Just $ "(coin." <> case _kadenaAddress_accountCreated recipient of
+          AccountCreated_Yes -> "transfer"
+          AccountCreated_No -> "transfer-create"
+        , Just $ tshow $ unAccountName $ _account_name sender
+        , Just $ tshow $ unAccountName $ _kadenaAddress_accountName recipient
+        , case _kadenaAddress_accountCreated recipient of
+          AccountCreated_Yes -> Nothing
```

Comment:
using `mempty` or `Text.empty` rather than having to go via `catMaybes` ?

Reply:
Nice